dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf

```diff
@@ -0,0 +1,29 @@
+resource "kafka_topic" "uswitch.data.v1" {
+  name = "customer_proposition.uswitch.data.v1"
+
+  replication_factor = 3
+  partitions         = 15
+
+  # infinte retention
+  config = {
+    "remote.storage.enable" = "true"
+    "retention.bytes" = "-1"
+    "retention.ms" = "-1"
+    "local.retention.ms" = "3600000"
+    "max.message.bytes" = "1048576"
+    "compression.type"  = "zstd"
+    "cleanup.policy"    = "delete"
+  }
+}
+
+module "uswitch-data-projector" {
```

Comment:
I assume if I later want to allow `uswitch-data-projector` to consume a topic as well, i can just add that here?

Reply:
Exactly